packages/cardano-services/test/jest-setup/jest-teardown.ts

```diff
@@ -1,7 +1,9 @@
+import { RabbitMQContainer } from '../../../rabbitmq/test/docker';
 import { removePostgresContainer } from './docker';
-import { removeRabbitMQContainer } from '../../../rabbitmq/test/jest-setup/docker';
 
 module.exports = async () => {
+  const container = new RabbitMQContainer();
```

Comment:
Do we need to create a new `RabbitMQContainer` container instance in order just to stop the same instance at line 8?

Reply:
After following the code changes, now I see the reason for using `new RabbitMQContainer` constructor is just to find the relevant `containerName` and try to stop it. You can ignore my question :)